pytest_catchlog.py

```diff
@@ -154,6 +175,72 @@ def pytest_runtest_teardown(self, item):
         with self._runtest_for(item, 'teardown'):
             yield
 
+    def configure_console_handler(self, config):
+        verbosity = config.getoption('-v')
+        # Prepare the handled_levels dictionary
+        log_levels = []
+        handled_levels = {}
+        available_levels = set([
+            #logging.CRITICAL,  # This will be the default console level if no -v is passed
+            logging.ERROR,
+            logging.WARNING,
+            logging.INFO,
+            logging.DEBUG,
+            logging.NOTSET
+        ])
+        for level in get_option_ini(config, 'log_extra_levels'):
+            try:
+                level_num = int(getattr(logging, level, level))
+            except ValueError:
+                # Python logging does not recognise this as a logging level
+                raise pytest.UsageError(
+                    '\'{0}\' is not recognized as a logging level name. Please '
+                    'consider passing the logging level num instead.'.format(
+                        level
+                    )
+                )
+            available_levels.add(level_num)
+
+        for level in available_levels:
+            if level > logging.WARN:
```

Comment:
Yeah, and I even think WARN is going to  be deprecated..... At least the method is....